addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -281,65 +281,49 @@
                 <div class="modal-content">
                     <div class="modal-header align-items-center">
                         <h4 class="modal-title">Attendees</h4>
-                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"/>
                     </div>
-                    <t t-set="counter_type" t-value="1"/>
-                    <t t-set="counter" t-value="0"/>
-                    <t t-foreach="tickets" t-as="ticket" t-if="availability_check">
-                        <t t-foreach="range(1, ticket['quantity'] + 1)" t-as="att_counter" name="attendee_loop">
-                            <t t-set="counter" t-value="counter + 1"/>
-                            <div class="modal-body border-bottom">
-                                <h5 class="mt-1 pb-2 border-bottom">Ticket #<span t-out="counter"/> <small class="text-muted">- <span t-out="ticket['name']"/></small></h5>
-                                <div class="row">
-                                    <div class="col-lg my-2">
-                                        <label>Name *</label>
-                                        <input class="form-control" type="text" t-attf-name="#{counter}-name" required="This field is required"
-                                            t-att-value="default_first_attendee.get('name', '') if counter == 1 else ''"/>
-                                    </div>
-                                    <div class="col-lg my-2">
-                                        <label>Email *</label>
-                                        <input class="form-control" type="email" t-attf-name="#{counter}-email" required="This field is required"
-                                            t-att-value="default_first_attendee.get('email', '') if counter == 1 else ''"/>
-                                    </div>
-                                    <div class="col-lg my-2">
-                                        <label>Phone <small>(Optional)</small></label>
-                                        <input class="form-control" type="tel" t-attf-name="#{counter}-phone"
-                                            t-att-value="default_first_attendee.get('phone', '') if counter == 1 else ''"/>
+                    <t t-if="availability_check">
+                        <t t-set="counter" t-value="0"/>
+                        <t t-set="input_type_by_question_type" t-value="{'name': 'text', 'email': 'email', 'phone': 'tel', 'company_name': 'text'}"/>
+                        <t t-foreach="tickets" t-as="ticket">
```

Comment:
Could probably keep the t-if inside the t-foreach as I believe this increases the diff by a lot? As we have an extra indentation level everywhere?